.devcontainer/scripts/container-creation.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install SDK and tool dependencies before container starts
+# Also run the full restore on the repo so that go-to definition
+# and other language features will be available in C# files
+./build.sh
```

Comment:
`./restore/sh` should be sufficient.

Reply:
Changed with latest commit! Thanks for feedback! :)